.travis.yml

```diff
@@ -8,6 +8,12 @@ cache:
         - $HOME/.cache/pip
         - $HOME/ossl-1/
 
+# Only build master and the version branches (e.g. 1.7.x)
+branches:
+    only:
+        - master
+        - /\d+\.\d+\.+x/
```

Comment:
Oops, from the docs it looks like `branches` also applies to `tags`, so please add a regex for them.